.github/workflows/std.yml

```diff
@@ -1,33 +1,23 @@
 # yq -y < std-unexploded.yml > workflows/std.yml
 name: STD
 on:
-  workflow_dispatch:
-    inputs:
-      deploy:
-        description: Deploy to dev-preview
-        type: boolean
-        required: true
-        default: false
```

Comment:
I think we should preserve this functionallity

Reply:
It complicates triggers, but okay, let me think.

Preserve it _only_ for `dev-preview`?

Reply:
Maybe this is something to discuss duringe the call starting in 1 hr

Reply:
We should have it for all enviornments but I am okay with keeping this only for dev-preview until future iterations

Reply:
I like the button, too, but I'm afraid we won’t be allowed to do this. :pensive:

Because even in `dapps` (which is of lower significance and not production code yet), Rhys wanted it removed, and do everything through PRs to deployment branches…

Reply:
Fixed in d3b5fdc8c479a58e0bfc041b474bae06dcbc0ab9